cli/src/commands/squash.rs

```diff
@@ -176,15 +177,24 @@ pub(crate) fn cmd_squash(
     )? {
         let mut commit_builder = squashed.commit_builder.detach();
         let new_description = match description {
-            SquashedDescription::Exact(description) => description,
+            SquashedDescription::Exact(description) => {
+                commit_builder.set_description(description);
+                add_trailers(ui, &tx, &commit_builder)?
+            }
             SquashedDescription::UseDestination => destination.description().to_owned(),
             SquashedDescription::Combine => {
                 let abandoned_commits = &squashed.abandoned_commits;
                 if let Some(description) = try_combine_messages(abandoned_commits, &destination) {
                     description
```

Comment:
Perhaps, we should also add trailers to combined `description` (n = 1) if it's not empty?

Reply:
I didn't do it to have the same behavior as when using `--use-destination-message`, but I'm not sure what the behavior should be.

Reply:
Maybe insert trailers to `UseDestination`, too? It seems odd that combined description is updated depending on the number of non-empty texts.